pkg/testmachinery/argo/argo.go

```diff
@@ -21,11 +21,16 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+// the label and taint of the nodes in the worker pool which are reserved for workflow pods
+const workerPoolTaintLabelName = "testload"
```

Comment:
I would move this constant to https://github.com/gardener/test-infra/blob/a939f23ebc0c12cb3018788bea385f860cd91505/pkg/testmachinery/config.go
then we have all configuration in one place and do not need to search for configuration in different places.

Reply:
Maybe also make it configurable from the outside via parameter or env var like [here](https://github.com/gardener/test-infra/blob/7b2ed5f6a3b78dd7de3b1e84bf74e6557eac62f9/pkg/testmachinery/testmachinery.go#L32)

Reply:
I moved the config.
I guess there's no need for making this feature configurable as workflow pods only _prefer_ nodes labeled with `purpose: testload`. If no such nodes exist, the workflow pods will happily schedule to whatever nodes are available.

Reply:
ok then this is ok